node_modules/oae-resource/lib/actions.js

```diff
@@ -589,7 +587,15 @@ var _getTargets = function(targetIds, callback) {
 
             return callback(null, targetsById);
         });
-    });
+    };
+
+    // For all email targets, see if there is already an account associated with them. If so, they
+    // will be pre-validated user share targets
+    if (!_.isEmpty(emails)) {
```

Comment:
Is there a reason you've replaced `OaeUtil.invokeIfNecessary` here? Just tidier I guess?